design/api_design.md

```diff
@@ -1,11 +1,10 @@
-GET:
+**GET**:
 
-* get_transaltion_id
-* get_texts_by_unit
-* get_suggested_text
-* get_translated_percent
-* get_languages
+* `api/translate?type=<>&lang=<>&pk=<>` - translate any type of object with given lang with pk
```

Comment:
@KirillMakarov  lesson, step. I suppose `en` or `en_ru`. I suppose they are the same with step.block.text. Attempts? Should we translat them?